doc/release_notes.md

```diff
@@ -2,6 +2,67 @@
 
 Descriptions of the tagged TKey releases.
 
+
+## TK1-24.03.xx
+
+This is a official release of the "Bellatrix" version of the Tillitis'
+TKey. This version is ready for general use.
+
+Given the OCI image `ghcr.io/tillitis/tkey-builder:4` built from
+`../contrib/Dockerfile` and the generic UDS.hex and UDI.hex, a clean
+build should generate the following digest:
+
+```
+5b373c0ab86b8970b28270bb54825e1dc7f9723a80598ad85851c2b3aabed989  application_fpga.bin
+```
+
+### FPGA:
+- Security Monitor now prevents access to RAM outside of the physical
+  memory. If it detects one it will halt the CPU.
```

Comment:
Wording: 'if it detects one' --> 'if it detects an access outside of the RAM address space'. 
Many more words, but 'one' is quite imprecise

Reply:
Fixed.